packages/project-disaster-trail/src/components/Game/TaskScreen.js

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import { PropTypes } from "prop-types";
+import { connect } from "react-redux";
+import { map } from "lodash";
+import {
+  getTaskOrder,
+  getActiveTask,
+  getActiveEnvironment,
+  getTasksForEnvironment,
+  doNextTask,
+  changeEnvironment
+} from "../../state/tasks";
+
+const TaskScreen = ({
+  taskOrder,
+  activeTask,
+  activeEnvironment,
+  tasksForEnvironment,
+  addTask,
+  updateEnvironment
+}) => {
+  const possibleTasks = [];
```

Comment:
This is really just preference, but I'd write this more simply using native Array.map:
```js
const possibleTasks = tasksForEnvironment[activeEnvironment].map(task => (
  <button
    key={task}
    type="button"
    onClick={() => {
      addTask(task);
    }}
  >
    {task}
  </button>
));
```